### ISSUE TITLE
Does GAPIT take usual covariate? like sex or other categorical covariate with 0s and 1s?

### ISSUE THREAD

**Issue by lzhangUT** [jiabowang/GAPIT#101]
Hi,
I am new to GAPIT, after reading the tutorial, I would like to give a try of GAPIT for my data. but my main goal here is to include a common covariate as my CV file into the analysis. but my CV file is not subpopulation structure or any PCs, just common covariate, can GAPIT take care of that?
Thanks

**Comment by jiabowang** [comment]
Yes, covariates can be taken in the GAPIT. But all values should be numeric with any missing values.
